components/Header/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return <StyledHeader>Hammerhart</StyledHeader>;
```

Comment:
At this point the Header component doesn't really make sense, because it doesn't include logic for now. 

It would be better to just add the StyledHeader directly to the index.js.

Do you want to add more logic to the header in the future? Then you can leave it for now. If  you don't its a bit over-engeneered :D